### PR TITLE
Updated to match indy crypto

### PR DESCRIPTION
**Pull request by mikelodder7**
This is to correspond to [Pull Request #105](https://github.com/hyperledger/indy-crypto/pull/105) at indy crypto

Signed-off-by: Michael Lodder <redmike7@gmail.com>

### CODE REVIEW

**Comment by mikelodder7** [comment]
I will remove the docket file. This is the one I use for testing my builds. I also have an android one but I’ll submit that in a different PR.

File: libindy/Cargo.toml [hunk 1]
```diff
@@ -20,8 +20,7 @@ path = "src/lib.rs"
 crate-type = ["staticlib","rlib", "dylib"]
 
 [features]
-default = ["bn_openssl", "box_sodium", "sealedbox_sodium", "base58_rust_base58", "xsalsa20_sodium", "pair_amcl", "hash_openssl", "local_nodes_pool", "revocation_tests"]
-bn_openssl = ["openssl", "int_traits"]
+default = ["box_sodium", "sealedbox_sodium", "base58_rust_base58", "xsalsa20_sodium", "pair_amcl", "hash_openssl", "local_nodes_pool", "revocation_tests"]
```
[the remark this answers is not in the document]
Comment (mikelodder7) [reply libindy/Cargo.toml]: So you want me to point to my repo for this pull request then submit another PR that undoes it once the PR for Indy-crypto is pulled?

[the remark this answers is not in the document]
Comment (jovfer) [reply libindy/Cargo.toml]: I suggest the next sequence:
1) temporary point your repo in this PR
2) finalize changes in sdk and crypto PRs
3) merge crypto
4) update this PR to new master (or even release?) version of crypto
5) merge this PR

**Comment by jovfer** [comment]
@mikelodder7 last CI failed for:
* libindy tests on ubuntu, amazonlinux and windows
* iOS, python and java wrappers

**Pull request merged**